src/SequenceManager.cc

```diff
@@ -327,16 +339,68 @@ void SequenceManager::rename_sequence(UniqueId unique_id, const SequenceName& ne
         throw Error(gul14::cat("Cannot rename folder ", old_path.string(),
             " to ", new_path.string(), ": ", error.message()));
     }
+
+    // commit to local repository
+    gl_.add_files({gul14::cat((const std::string&) new_path, "/*")});
```

Comment:
Please do not use C-style casts (see [Core Guidelines ES.49](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es49-if-you-must-use-a-cast-use-a-named-cast)). Here, the same effect can be probably be had by calling `.string()`.

Reply:
removed all occurences